### DISCUSSION TITLE
可以用上一次训练出的模型 当作底模进行下一炉训练吗

### DISCUSSION THREAD

**Discussion by fish5257** [svc-develop-team/so-vits-svc#296] [Q&A]
其实想问怎么制作自己的底模

**Comment by KakaruHayate** [comment]
这是可以的，一般来讲重新进行数据预处理后都可以把之前的模型作为预训练模型继续训练，大多数情况下比直接训练要好

**Comment by HuanLinOTO** [comment]
it's look lite a problem of using, concert to discussion
